crates/pop-cli/src/common/builds.rs

```diff
@@ -55,6 +54,43 @@ pub fn ensure_node_binary_exists(
 	}
 }
 
+/// Creates a chain specification builder based on project structure.
+///
+/// # Arguments
+/// * `path` - Path to the project.
+/// * `profile` - Build profile to use.
+/// * `default_bootnode` - Whether to use default bootnode.
+/// * `cli` - Command line interface implementation.
+///
+/// # Returns
+/// The chain spec builder for the node or the runtime.
+#[cfg(feature = "chain")]
+pub fn create_chain_spec_builder(
+	path: &Path,
+	profile: &Profile,
+	default_bootnode: bool,
+	cli: &mut impl Cli,
+) -> anyhow::Result<ChainSpecBuilder> {
+	let default_node_path = path.join("node");
```

Comment:
Is this folder always has to be called `node`?

Reply:
Yes, indeed.